client/state/lib/README.md

```diff
@@ -0,0 +1,75 @@
+Library Middleware
+===========
```

Comment:
Super nitpick-y, but number of `=`s should match no. of chars right above :slightly_smiling_face: